### PR TITLE
SVCS-2561 Use different package for hmac sign

### PR DESCRIPTION
**Pull request by AaronTrippaers**

### CODE REVIEW

File: src/sha1.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ */
+
+/**
+ * Computes a HMAC-SHA1 code.
+ *
+ * @param {string} k Secret key.
+ * @param {string} d Data to be hashed.
+ * @return {string} The hashed string.
+ */
+export function hmacsha1(k, d, _p = undefined, _z = undefined) {
```
Comment (AaronTrippaers) [inline src/sha1.ts:13]: The readable boi did not want to work. 
Added the export, renamed to function added to `=undefined` to the `_p` and `_z` parameters.
Prettier did the formatting.

File: src/sha1.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+ */
+
+/**
+ * Computes a HMAC-SHA1 code.
+ *
+ * @param {string} k Secret key.
+ * @param {string} d Data to be hashed.
+ * @return {string} The hashed string.
+ */
+export function HmacSHA1(k, d, _p = undefined, _z = undefined) {
```
Comment (AaronTrippaers) [inline src/sha1.ts:13]: The readable boi did not want to work.
Added the export, renamed to function added to =undefined to the _p and _z parameters.
Prettier did the formatting.

**Review by chad-pelly — DISMISSED** [review]
Nailed it! ⚔️ 

What's with the removal of the Jest stuff in this PR? 🤔

**Comment by AaronTrippaers** [comment]
> Nailed it! ⚔️
> 
> What's with the removal of the Jest stuff in this PR? 🤔

The stuff that was removed from jest was just there because es-crypto, which is now removed.
https://github.com/ExtraHorizon/javascript-sdk/pull/848/files#diff-759a02c8fa99d555891bb2a2d3ae2495a2295dc1e1183dc8ba6281b0e172208a

File: src/sha1.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
+/* eslint-disable */
+/**
+ * Copied from https://github.com/pH200/hmacsha1-js/blob/master/index.js
```
Comment (wwaaijer-exh) [inline src/sha1.ts:3]: Can you also paste the full copyright message here? Would be appropriate I belief (for when the link is dead for instance)

Comment (AaronTrippaers) [reply src/sha1.ts:3]: Done ✅

**Pull request merged**